.travis.yml

```diff
@@ -195,7 +195,7 @@ install:
       # To run a PR with a patched phpunit-bridge, first submit the path for the
       # phpunit-bridge as a separate PR against the next feature-branch then
       # uncomment and update the following line with that PR number
-      #SYMFONY_PHPUNIT_BRIDGE_PR=32886
+      SYMFONY_PHPUNIT_BRIDGE_PR=ticket_33592
```

Comment:
This change is wrong:

- `ticket_33592` is not a valid PR number
- this PR is a PR editing the bridge itself, not a PR when you want to use a pending PR on the bridge to run tests, so there is no reason to use that feature.